.github/workflows/test-js-packages.yml

```diff
@@ -595,10 +595,23 @@ jobs:
           sed -i '/^compiler_version/d' ./**/Nargo.toml
       
       - name: Run nargo test
+        id: test_report
         working-directory: ./test-repo/${{ matrix.project.path }}
         run: |
+
           output_file=${{ github.workspace }}/noir-repo/.github/critical_libraries_status/${{ matrix.project.repo }}/${{ matrix.project.path }}.actual.jsonl
+          BEFORE=$SECONDS
           nargo test --silence-warnings --skip-brillig-constraints-check --format json ${{ matrix.project.nargo_args }} | tee $output_file
+          TIME=$(($SECONDS-$BEFORE))
+
+          NAME=${{ matrix.project.repo }}/${{ matrix.project.path }}
+          # Replace any slashes with underscores
+          NAME=${NAME//\//_}
+          TEST_REPORT_NAME=test_report_$NAME
+          echo "test_report_name=$TEST_REPORT_NAME" >> $GITHUB_OUTPUT
+
+          jq --null-input "{ test_reports: [{ name: \"$NAME\", value: (\"$TIME\" | tonumber), unit: \"s\" }]}" > $TEST_REPORT_NAME.json
```

Comment:
```suggestion
          echo "{ \"test_reports\": [{ \"name\": \"$NAME\", \"value\": $TIME, \"unit\": \"s\" }]}" > $TEST_REPORT_NAME.json
```
Do we need jq?

Reply:
As things are working though feel free to clean this up later

Reply:
I quite like how we can avoid the need to perform so many escapes tbh. This seems much more readable to me.